lib/internal/fs/cp/cp.js

```diff
@@ -313,13 +313,16 @@ async function mkDirAndCopy(srcMode, src, dest, opts) {
 
 async function copyDir(src, dest, opts) {
   const dir = await opendir(src);
+  const promises = [];
 
   for await (const { name } of dir) {
     const srcItem = join(src, name);
     const destItem = join(dest, name);
     const { destStat, skipped } = await checkPaths(srcItem, destItem, opts);
-    if (!skipped) await getStatsForCopy(destStat, srcItem, destItem, opts);
+    if (!skipped) PromisePrototypePush(promises, getStatsForCopy(destStat, srcItem, destItem, opts));
```

Comment:
Oops sorry, I think I was tired
```suggestion
    if (!skipped) ArrayPrototypePush(promises, getStatsForCopy(destStat, srcItem, destItem, opts));
```